mathesar/exception_handlers.py

```diff
@@ -1,9 +1,23 @@
 from rest_framework.views import exception_handler
 
+from mathesar.api.exceptions.error_codes import ErrorCodes
+
 exception_map = {
 }
 
 
+def fix_error_response(data):
```

Comment:
I think `prettify_response` would be a better name here.

Reply:
There are two types of response data which are not formatted correctly.
1. Response data which does not have a proper structure
2,  Response data which has proper structure but has some quirks due to `drf` way of handling exceptions.

In case of (2) we are fixing the pretty error. Moreover we are checking if the error is pretty in previous condition block, so it would be conflicting to name it `pretty`,

Reply:
Can you name it `standardize_error_response` then? I think it's more descriptive.